docs/slides/aws/hcp-terraform-v2/index.html

```diff
@@ -7,7 +7,7 @@
 <html>
 
 <head>
-  <title>Terraform Cloud on GCP</title>
+  <title>HCP Terraform on AWS</title>
```

Comment:
Should this be AWS or GCP? It was changed from GCP

Reply:
The file path is for AWS, so it should be good as is.